src/main/java/com/revature/services/impl/TripServiceImpl.java

```diff
@@ -0,0 +1,57 @@
+package com.revature.services.impl;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.revature.models.Trip;
+import com.revature.repositories.TripRepository;
+import com.revature.services.TripService;
+import org.springframework.stereotype.Service;
+
+@Service
+public class TripServiceImpl implements TripService {
+	@Autowired
+	private TripRepository tripRepository;
+
+	@Override
+	public List<Trip> getTrips() {
+		return tripRepository.findAll();
+	}
+
+	@Override
+	public Trip getTripById(int id) {
+		return tripRepository.findById(id).get();
```

Comment:
I see that most other services structure the getById method the same way, but we should consider refactoring them. If this ever gets called with an id that does not exist, it will throw an exception, which is not ideal.

Reply:
I agree we should refactor every method that could throw an exception. For now, refactored only TripRepository's findById to return an Optional.